app/controllers/api/v1/greetings_controller.rb

```diff
@@ -0,0 +1,10 @@
+class Api::V1::GreetingsController < ApplicationController
+  def index
+    @greetings = Greeting.all
+    if @greetings.empty?
+      json_response({ msg: 'No Greetings at this time' }, 400)
```

Comment:
- [ ] kindly ensure that you write the correct syntax of this method because it's giving an error that prevents the app from showing text that they are no messages at the time. you got this 👍 

the error :
![image](https://user-images.githubusercontent.com/90524466/187977869-1e995c22-c6c5-408d-ac16-2dd6b6e4b908.png)